.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+  - "2.7"
+env:
+  - TOX_ENV=py26
```

Comment:
Good to know. It might not matter whats defined in `tox.ini` as long as it doesnt need any env-specific dependencies.